enhancements/20220726-vm-migration.md

```diff
@@ -0,0 +1,109 @@
+# VM Migration controller
+
+## Summary
+
+To improve adoption we need to provider users an easy way to migrate VM workloads from other cloud providers such as 
+* vsphere
+* openstack
+
+### Related Issues
+
+https://github.com/harvester/harvester/issues/2274
+
+## Motivation
+
+### Goals
+We need a way to allow users to browse their VM workloads running on vmware, openstack or harvester clusters and migrate them to current harvester cluster.
+
+Currently there is no easy way to migrate VM's from other providers to harvester.
+
+### Non-goals [optional]
+
+
+## Proposal
+
+### User Stories
+
+#### Migrate a VM from Vsphere/Openstack/Harvester cluster
+* A user should be able to define a source vsphere/openstack or harvester cluster
+* The user should be able to browse the cluster to identify a VM, and configure its network mapping to harvester VLAN definitions.
```

Comment:
Would be better to specify what will be copied from the existing VM to the Harvester cluster, e.g, name, description, CPU, Memory, network config, the cloud-init template only, or even less?

Reply:
the cpu, memory and network config is copied, along with all attached disks.

iso disks are not copied across.

Reply:
I will update the HEP with more information.